pyccel/ast/cwrapper.py

```diff
@@ -528,12 +548,56 @@ def pybool_to_bool(cast_function_name):
                        body      = cast_function_body,
                        results   = [cast_function_result])
 
+def pyarray_to_ndarray(cast_function_name):
+    cast_function_argument = Variable(dtype=PyccelPyArrayObject(), name='o',
+                                is_pointer=True)
+    cast_function_result   = Variable(dtype=NativeInteger(), name = 'c',
+                                rank=1)
```

Comment:
Is it really necessary to split these two lines? See comment below

Reply:
I am trying to respect the rule of pep.
the line should not contain more than 79 characters.

Reply:
The rule is there to help you write better code, not worse!
How long are these lines?
```python
Arr = PyccelPyArrayObject()
Int = NativeInteger()

cast_function_argument = Variable(Arr, 'o', is_pointer=True)
cast_function_result   = Variable(Int, 'c', rank=1)
```